MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1130,19 +1130,30 @@ public string GetVideoQualityParam(EncodingJobInfo state, string videoEncoder, E
             var profile = state.GetRequestedProfiles(targetVideoCodec).FirstOrDefault();
             profile =  Regex.Replace(profile, @"\s+", String.Empty);
```

Comment:
```suggestion
            var profile = state.GetRequestedProfiles(targetVideoCodec).FirstOrDefault() ?? string.Empty;
            profile = Regex.Replace(profile, @"\s+", string.Empty);
```

Fixes the exception from https://github.com/jellyfin/jellyfin-roku/issues/341